randovania/gui/ui_files/games_tab_am2r_widget.ui

```diff
@@ -132,15 +132,15 @@
; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;EMP puzzles don't unlock all doors in the room anymore, but only relevant ones.&lt;/li&gt;&lt;li style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;The door to Plasma Chamber will now be blue after defeating Tester, instead of red.&lt;/li&gt;&lt;li style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Larva Metroids will now always give ammo drops when necessary, instead of having a small chance to drop nothing.&lt;/li&gt;&lt;li style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;The file selection screen will now show a hash of the current seed.&lt;/li&gt;&lt;li style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;The Item in Gravity Chamber will now always be in the open, with the Gravity Pod cutscene being disabled.&lt;/li&gt;&lt;li style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;The bomb blocks in &amp;quot;The Tower - Tower Exterior North&amp;quot; will now always be gone if you come from the &amp;quot;Dark Maze&amp;quot; room.&lt;/li&gt;&lt;li style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;When going down the Elevator from GFS Thoth, the Power Bomb Blocks will be automatically gone if you haven't cleared them before, as otherwise you'd be stuck.&lt;/li&gt;&lt;li style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Long Range Activation has been enabled for every room, allowing for more Zip strats.&lt;/li&gt;&lt;li style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;In GFS Thoth, both sides on the &amp;quot;Thoth Bridge&amp;quot; now have doors added for better Door Lock Randomizer compatibility.&lt;/li&gt;&lt;li style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;In Hydro Station, the connection from &amp;quot;Water Turbine Station&amp;quot; to &amp;quot;Hydro Station Exterior&amp;quot; now has a door for better Door Lock Randomizer compatibility.&lt;/li&gt;&lt;li style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;The Geothermal Power Plant is always in its exploded state. The Power Bomb Expansion that was located there has been moved to &amp;quot;Power Plant Destroyed Shaft&amp;quot;.&lt;/li&gt;&lt;/ul&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
"Their location can be found in the FAQ." -> maybe "their location**s**"? since right before it mentions septoggs in plural